### PR TITLE
test for back button action

### PR DESCRIPTION
**Pull request by jpita**

### CODE REVIEW

File: cypress/integration/spec.js [hunk 1]
```diff
@@ -36,4 +36,16 @@ describe('Article search', () =>{
...
+  it('back button should take us to search page with results', () => {
+    searchPage.search("cattle")
+    searchPage.results().first()
+    cy.enter()
+    cy.downArrow()
+    cy.enter()
+    articlePage.title().should('have.text', "Cattle")
+    cy.clickCloseButton()
+    searchPage.results().first()
+      .children().first().next().should('have.class', 'img')
```
Comment (hueitan) [inline cypress/integration/spec.js]: how do we tell having the class img belongs to search page with results?

Comment (jpita) [reply cypress/integration/spec.js]: I'm sorry, I don't understand the question

**Comment by jpita** [comment]
> The invisionapp sketch shows the `left / center / right` are `menu / selection / sections`
> 
> I think there is no close button in this view, therefore the default back button is needed.

Are you requesting a change in the app's code or the test?

**Comment by hueitan** [comment]
oh wait, zeplin has close button on the left, let's get clarify from design.

File: cypress/integration/spec.js (hunk 1 above)
Comment (hueitan) [reply cypress/integration/spec.js]: I'm not very sure, let's resolve this.

**Pull request merged**